.github/workflows/validation.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.17.2'
+          go-version: '1.19.3'
```

Comment:
Just a reminder to switch to 1.19.4 when it's available.

Reply:
1.19.4 was a general reminder. We have to update the notice also but there is another comment about this.